app/views/katello/api/v2/content_views/base.json.rabl

```diff
@@ -39,6 +43,12 @@ end
 
 child :environments => :environments do
   attributes :id, :name, :label
+  node :activation_keys do |env|
+     @resource&.activation_keys&.in_environment(env)&.ids
+  end
+  node :hosts do |env|
+     @resource&.hosts&.in_environment(env)&.ids
+    end
```

Comment:
@resource is not being set in the environment child scope for the base rabl when coming from index template. It is set when coming from show template. 
Trying to debug that. For now, environments node has the activation key and hosts Ids set only in CV details API call.